lib/hls/manifest_text_parser.js

```diff
@@ -192,27 +192,30 @@ shaka.hls.ManifestTextParser.prototype.parseTag_ = function(word) {
  */
 shaka.hls.ManifestTextParser.parseTag = function(id, word) {
   /* HLS tags start with '#EXT'. A tag can have a set of attributes
-    (#EXT-<tagname>:<attribute list>) or a value (#EXT-<tagname>:<value>).
+    (#EXT-<tagname>:<attribute list>) and/or a value (#EXT-<tagname>:<value>).
     An attribute's format is 'AttributeName=AttributeValue'.
     The parsing logic goes like this:
      1. Everything before ':' is a name (we ignore '#').
-     2. Everything after should be parsed as attributes if it contains '='.
-     3. Otherwise, this is a value.
+     2. Everything after ':' is a list of items,
```

Comment:
This is confusingly worded. I think it'd be clearer like...

2. Everything after ':' is a list of comma-separated items.
2a. The first item might be a value, if it does not contain '='.
2b. Otherwise, items are attributes.

Reply:
Yeah, it was better originally but exceeded the 80 character limit, then it went downhill. Copy-pasted your wording. Sounds much better.